etl/steps/data/garden/who/2023-08-04/icd_codes.py

```diff
@@ -0,0 +1,45 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+from typing import cast
+
+import owid.catalog.processing as pr
+from owid.catalog import Dataset
+
+from etl.data_helpers import geo
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load meadow dataset.
+    ds_meadow = cast(Dataset, paths.load_dependency("icd_codes"))
+
+    # Read table from meadow dataset.
+    tb = ds_meadow["icd_codes"]
+    tb = tb.reset_index()
+    icd_codes_map = {"Icd10": "ICD-10", "Icd9": "ICD-9", "Icd8": "ICD-8", "Icd7": "ICD-7"}
+    tb["icd"] = tb["icd"].replace(icd_codes_map)
+
+    # Calculate sum of number of countries using each ICD code type each year
+    tb_sum = tb.groupby(["year", "icd"]).count().reset_index()
+    tb_sum = tb_sum.rename(columns={"icd": "country", "country": "countries_using_icd_code"})
+
+    # More of a traditional grapher dataset. For each country-year, which ICD code is being used.
+    tb = geo.harmonize_countries(df=tb, countries_file=paths.country_mapping_path)
+
+    # Combine the datasets
+    tb_combined = pr.concat([tb_sum, tb])
```

Comment:
Also, why don't you just create two different tables for the ICD totals and the country ICDs? It might be confusing to see two different types of entities in `country`

Reply:
Ah good point, I think I tried it and had some issues in only being able to upload one table to grapher, but you are right I can have two in garden and then combine them in grapher.

Reply:
And how about uploading two tables in Grapher? I think it should be fine

Reply:
I shall try, for some reason it seemed not to work before but perhaps that was because of another issue.

Reply:
You were right :)